frontend/src/pages/ProfileEditPage.tsx

```diff
@@ -0,0 +1,450 @@
+  };
+
+  const saveRunnerProfile = () => {
+    if (!isThereModification) return alert('수정된 내용이 없습니다');
+
+    if (!runnerProfile) return;
+
+    if (!name) return alert('이름을 입력해주세요');
+    if (!company) return alert('소속을 입력해주세요');
+    if (!introduction) return alert('자기소개를 입력해주세요');
+    if (!technicalTags) return;
+
+    const newInfo = { name: name.trim(), company: company.trim(), introduction: introduction.trim(), technicalTags };
+
+    setRunnerProfile({ ...runnerProfile, ...newInfo });
+    updateProfileInputValue({ ...runnerProfile, ...newInfo });
+    patchRunnerProfile(newInfo);
+  };
+
+  const saveSupporterProfile = () => {
+    if (!isThereModification) return alert('수정된 내용이 없습니다');
+
+    if (!supporterProfile) return;
+
+    if (!name) return alert('이름을 입력해주세요');
+    if (!company) return alert('소속을 입력해주세요');
+    if (!introduction) return alert('자기소개를 입력해주세요');
+    if (!technicalTags) return;
+
+    const newInfo = { name: name.trim(), company: company.trim(), introduction: introduction.trim(), technicalTags };
+
+    setSupporterProfile({ ...supporterProfile, ...newInfo });
+    updateProfileInputValue({ ...supporterProfile, ...newInfo });
+    patchSupporterProfile(newInfo);
+  };
+
+  const confirmTagSelect = (tags: TechnicsType[]) => {
+    setTechnicalTags([...tags]);
+    closeModal();
+  };
+
+  const handleClickRunnerButton = () => {
+    if (!runnerProfile) return alert('러너 프로필 정보를 불러오지 못했습니다');
+
+    if (isThereModification) {
+      if (confirm('서포터 프로필에 수정 사항이 있습니다. 최근 편집된 내용을 저장할까요?')) {
+        saveSupporterProfile();
+      }
+    }
+
+    setIsRunner(true);
+    updateProfileInputValue(runnerProfile);
+  };
+
+  const handleClickSupporterButton = () => {
+    if (!supporterProfile) return alert('서포터 프로필 정보를 불러오지 못했습니다');
+
+    if (isThereModification) {
+      if (confirm('러너 프로필에 수정 사항이 있습니다. 최근 편집된 내용을 저장할까요?')) {
+        saveRunnerProfile();
+      }
+    }
+
+    setIsRunner(false);
+    updateProfileInputValue(supporterProfile);
+  };
+
+  const openModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setIsModalOpen(false);
+  };
+
+  const patchRunnerProfile = async (runnerProfile: RunnerProfileRequest) => {
+    const token = getToken()?.value;
+    if (!token) {
+      return alert('토큰이 존재하지 않습니다');
+    }
+
+    const body = JSON.stringify(runnerProfile);
+    const authorization = `Bearer ${token}`;
+
+    await patchRequest<RunnerProfileRequest>(`/profile/runner/me`, authorization, body);
+  };
+
+  const patchSupporterProfile = async (supporterProfile: SupporterProfileRequest) => {
+    const token = getToken()?.value;
+    if (!token) {
+      return alert('토큰이 존재하지 않습니다');
+    }
+
+    const body = JSON.stringify(supporterProfile);
+    const authorization = `Bearer ${token}`;
+
+    patchRequest<SupporterProfileRequest>(`/profile/supporter/me`, authorization, body);
+  };
+
+  useEffect(() => {
+    const fetchRunnerProfile = async () => {
```

Comment:
```suggestion
    const getRunnerProfile = async () => {
```
위에는 patch라는 http메서드명을 썼는데, 여기는 fetch네요??
get이 어떤가요?

Reply:
이전에 지었던 함수명을 그대로 썼네요... 수정하겠습니당